src/constants/platform/android.tsx

```diff
@@ -0,0 +1,123 @@
+import { APPLY_ANDROID_PAGE, FAQ_ANDROID_PAGE, RECRUIT_ANDROID_PAGE } from '@/constants';
+
+const android = {
+  name: 'Android',
+  role: 'Android Developer',
+  path: {
+    recruit: RECRUIT_ANDROID_PAGE,
+    faq: FAQ_ANDROID_PAGE,
+    apply: APPLY_ANDROID_PAGE,
+  },
+  introduction: (
+    <>
+      Mash-Up 안드로이드팀은 세미나, 프로젝트, 스터디 등 여러 활동을 진행합니다.
+      <br />
+      학생부터 현업 직장인까지 다양한 연령대의 사람들이 모여 있으며 좋은 지식을 다 같이 공유하고
+      성장하며 서로에게 자극이 되기도 하는 관계를 지향합니다. 개발 뿐만 아니라 안드로이드 개발이라는
+      같은 공통사 가진 좋은 사람들과 만날 수 있는 친목의 장이 되기도 합니다.
+      <br />
+      안드로이드팀은 열정있는 안드로이드 개발자분들을 기다리고 있습니다!
+    </>
```

Comment:
br태그 말고 fragment안에 {''}로 텍스트를 감싸서 \n을 주는 방법은 안될까요? 지금 피드백을 달라는건 아니고 머지후에 추후에 br을 빼줄 수 있는 방법이 있다면 적용해보면 좋을거 같아요 :)

```suggestion
    <>
      {'Mash-Up 안드로이드팀은 세미나, 프로젝트, 스터디 등 여러 활동을 진행합니다.
      \n \n
      학생부터 현업 직장인까지 다양한 연령대의 사람들이 모여 있으며 좋은 지식을 다 같이 공유하고
      성장하며 서로에게 자극이 되기도 하는 관계를 지향합니다. 개발 뿐만 아니라 안드로이드 개발이라는
      같은 공통사 가진 좋은 사람들과 만날 수 있는 친목의 장이 되기도 합니다.
      \n \n
      안드로이드팀은 열정있는 안드로이드 개발자분들을 기다리고 있습니다!'}
    </>
```

Reply:
[white-space](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) 속성과 함께 사용해야 할 것 같은데 추후에 한번 일괄적으로 변경해볼게요!